udata_front/theme/gouvfr/assets/js/components/AdminTable/Table/AdminTableTh.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <th class="fr-text--bold">
+    <slot v-if="!sortable"></slot>
+    <div class="fr-grid-row fr-grid-row--middle justify-between" v-else>
+      <slot></slot>
+      <button
+        class="fr-ml-1w fr-btn fr-btn--sm fr-btn--tertiary-no-outline"
+        :class="{
+          'fr-icon-arrow-down-line': !sorted || sorted === 'asc',
+          'fr-icon-arrow-up-line': sorted === 'desc',
+          'text-grey-500': sorted,
+          'text-mention-grey': !sorted,
+        }"
+        @click="toggleSort"
+        :title="t('Sort')"
+      >
+        {{ t("Sort") }}
+      </button>
+    </div>
+  </th>
+</template>
+
+<script lang="ts">
+import type { SortDirection } from "../../../types";
+export type AdminTableThProps = {
+  sortable?: boolean;
```

Comment:
Do we need this prop? Could we just have a:
```
const sortable = computed(() => props.sorted !== undefined)
``` 

It's only working if `sorted?: SortDirection | null` and we return `null` in my previous comment about not returning `undefined` so we can differentiate between not passing the prop (not sortable) and passing the prop as `null` (sortable but not currently the sort column)